I.write-tests/03-remove-vowels-from-array/remove-vowels-in-array.test.js

```diff
@@ -1,11 +1,14 @@
+const removeVowelsForWords = require('./remove-vowels-in-array')
 test('remove vowels from all words in array', function () {
-    // Arrange
-
-    // Act
-
-    // Assert
+  // Arrange
+  const myArray = ['Irina', 'Etza', 'Daniel']
+  const expectation = ['rn', 'tz', 'dnl']
```

Comment:
Are you sure this is the correct expecation? Does the removeVowels function also lowercases the constenants?

Reply:
oppps corrected thanks